src/olympia/pages/templates/pages/shield_study_11.html

```diff
@@ -0,0 +1,59 @@
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_11/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Plugin Safety</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/plugin-safety/?src=shield-study-11"><button class="button primary large">Start Now</button></a>
+        </div>
+        <div class="content"><p>Mozilla wants to figure out the best way to keep our users safe if they choose to use third-party software such as plugins. Participating in this study will help us find out!</p>
+<h2 id="what-will-happen-next">What will happen next</h2>
+<p><strong>If you agree</strong> to participate:</p>
+<ol>
+<li>We may add security and plugin control features to Firefox to help with online security and performance.</li>
+<li>Continue using Firefox as you normally would. Depending on how you use the Web, you may not notice any changes.</li>
+<li>After a few weeks, the study will end on its own. Your Firefox will go back to normal.</li>
+<li>We will ask you to fill out a quick optional survey to tell us about your experience. The survey should take less than 5 minutes to complete.</li>
+</ol>
+<h2 id="leaving-the-study">Leaving the study</h2>
+<p>The study will expire on its own in a few weeks. You may leave the study early. To do so, follow these steps:</p>
+<ol>
+<li>Type about:addons into the location bar, and press enter.</li>
+<li>Find Plugin Safety in the addons list.</li>
+<li>Click the remove button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h2 id="your-privacy">Your privacy</h2>
+<h3 id="all-shield-studies">All Shield Studies</h3>
+<p>Every Shield Study collects data about important study events, such as install, uninstall, daily usage, and end of study. Studies also include an optional survey.</p>
+<p>This data is associated with <a href="https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Firefox Telemetry collection</a>.</p>
+<h3 id="this-particular-study">This particular Study</h3>
+<p>In addition to the data collected by all Shield Studies, Mozilla will collect information about: </p>
+<ul>
+<li>Websites you visit that try to use plugins, and what content on the site requires the plugin;</li>
+<li>How you interact with plugin content on those sites, including whether and how you choose to allow the plugin to run.</li>
+</ul>
+<p>Mozilla will keep this information private; any data will only be shared in aggregate or anonymized form.</p>
+<p>You can learn more about the data collection for this study <a href="https://github.com/squarewave/shield-study-essential-flash/blob/master/docs/metrics.md">here</a>.</p>
```

Comment:
There it is again, can we have a more reasonable name for the link instead of `here` please? Something like "on GitHub" or "in our Documentation" or so. That'll make it more accessibile to people.